src/views/Recorder/EmptyState.tsx

```diff
@@ -32,9 +40,15 @@ const RecorderEmptyStateSchema = z.object({
 type RecorderEmptyStateFields = z.infer<typeof RecorderEmptyStateSchema>
 
 export function EmptyState({ isLoading, onStart }: EmptyStateProps) {
+  const { data: settings } = useSettings()
+
   const proxyStatus = useProxyStatus()
   const { data: isBrowserInstalled } = useBrowserCheck()
 
+  const [captureBrowser, setCaptureBrowser] = useState(true)
```

Comment:
I didn't use `react-form-hooks` here because doing so with the `Checkbox` control was just way to much work for something so simple.